samples/ChatbotExamplev2/README.md

```diff
@@ -0,0 +1,111 @@
+
+## Set up Personalizer
+- In [appsettings.json](./appsettings.json), modify `PersonalizerServiceEndpoint` to be your resource's endpoint and `PersonalizerAPIKey` to be either one of your resource's keys; this can be found in the Keys and Endpoint tab in the portal under `Endpoint` and `Key1`/`Key2`, respectively.
+
+## Set up LUIS
+- Navigate to [LUIS portal](https://www.luis.ai).
+- Click the `Sign in` button.
+- Click on `My Apps`.
+- Select the Subscription for your LUIS authoring resource and then the LUIS authoring resource.
+- Click on the `New app for conversation` button.
+- Click on `Import as JSON` and select [coffeebot.json](./CognitiveModels/coffeebot.json) from the `cognitive-services-personalizer-samples/samples/ChatbotExample/CognitiveModels` folder.
+
+- Click on the [`Train`](https://docs.microsoft.com/en-us/azure/cognitive-services/luis/luis-how-to-train) button on the top right of the page; this will train your app on the model inside of chatbot.json.
+
+- Once this is done, press the [`Publish`](https://docs.microsoft.com/en-us/azure/cognitive-services/luis/luis-how-to-publish-app) button and select "Production" (You can leave the Sentiment Analysis, Spell Check, and Speech Priming fields set to Off). NOTE: Once you publish your app on LUIS portal for the first time, it takes some time for the endpoint to become available, about 5 minutes of wait should be sufficient.
+
+- Update [appsettings.json](./appsettings.json) with your `LuisAppId`, `LuisAPIKey`, and `LuisAPIServiceEndpoint`, all of which can be found under the `Manage` tab for your resource:
+    - `LuisAppId` can be found under `Settings`
+    - `LuisAPIKey` can be found under `Azure Resources` > `Prediction Resources` listed as `Primary Key`
+    - `LuisServiceEndpoint` can be found under the `Azure Resources` > `Prediction Resources` listed as `Endpoint URL`.  
+- You should now have an `appsettings.json` that looks like this:
+    ```json
+    {
+        "LuisAppId": "<your app ID>",
+        "LuisAPIKey": "<your LUIS API key>",
+        "LuisServiceEndpoint": "<your hostname, e.g. https://westus.api.cognitive.microsoft.com>",
+        "PersonalizerServiceEndpoint": "<your Personalizer endpoint, e.g. https://myPersonalizerResource.cognitiveservices.azure.com/",
+        "PersonalizerAPIKey": "<your Personalizer API key>"
+    },
+    ```
+
+# Running and interacting with the bot
+
+## Visual Studio
+- Press the `F5` key to run the project
+
+## Visual Studio Code
+- Bring up the terminal, navigate to `cognitive-services-personalizer-samples/samples/ChatbotExample/` folder.
+- Type `dotnet run`.
+
+## Connect to bot using Bot Framework Emulator
+- Launch Bot Framework Emulator
+- File -> Open Bot
+- Enter \<your App URL\>/api/messages as the Bot URL (default App URL for this project is `http://localhost:3978`)
```

Comment:
The post listed in this URL of 3978 isn't the port the chat bot is using.  It is using 44376 which I see in the cognitive-services-personalizer-samples\samples\ChatbotExamplev2\Properties\launchSettings.json file.

Reply:
changed to instruct the user how to find their app url